Dockerfile

```
@@ -21,6 +20,13 @@ ENV        PROGRESS=true
 COPY       ./env-config.sh /tmp/env-config.sh
 RUN        bash /tmp/env-config.sh
 
+
+# Install our package manager
+ARG npm_version=6.13.4
+ENV NPM_VERSION $npm_version
```

Do we want this to be configurable at build time? If not, we can omit the `ARG` and hardcode the `ENV`.

```suggestion
ENV NPM_VERSION 6.13.4
```